libdrgn/program.c

```diff
@@ -860,6 +901,17 @@ drgn_program_cache_core_dump_notes(struct drgn_program *prog)
 		assert(it.entry);
 		prog->crashed_thread = it.entry;
 	}
+
+	if (found_main_thread) {
```

Comment:
This should be `if (!prog->flags & DRGN_PROGRAM_IS_LINUX_KERNEL)` like `crashed_thread`.

Reply:
Fixed